test/cpp/util/grpc_tool.cc

```diff
@@ -72,6 +75,7 @@ DEFINE_string(proto_path, ".", "Path to look for the proto file.");
 DEFINE_string(protofiles, "", "Name of the proto file.");
 DEFINE_bool(binary_input, false, "Input in binary format");
 DEFINE_bool(binary_output, false, "Output in binary format");
+DEFINE_bool(repeated, false, "Wrap binary messages into parent protobuf message with tag number 1");
```

Comment:
Please format the change as clang-format suggested [here](https://grpc-testing.appspot.com/job/gRPC_pull_requests_linux/5657/testReport/junit/(root)/sanity_linux_dbg_native/tools_distrib_clang_format_code_sh/).